### DISCUSSION TITLE
[Feature] <Chinese support for web pages>

### DISCUSSION THREAD

**Discussion by lihaqi** [immich-app/immich#6383] [Feature Request]
### The feature

I am currently using the latest version 1.92.1, but the webpage does not support Chinese. I hope to consider supporting it.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
We currently don't have translation support for the web yet; it is only on the mobile app. Something that we are looking to do in the near future.

**Comment by bo0tzz** [comment]
Duplicate of #1688
